hack/traffic_flow_tests.sh

```diff
@@ -15,14 +15,6 @@ if [ -z "$worker" ]; then
   exit 1
 fi
 
-export KUBECONFIG=/root/kubeconfig.microshift
-nodes=$(oc get nodes)
-export acc=$(echo "$nodes" | grep -oP '^\d+-acc')
-if [ -z "$acc" ]; then
-  echo "Error: acc is empty"
-  exit 1
-fi
-
```

Comment:
Nice catch. This is now taken care of by the setup.sh script and indeed we don't need this anymore